site/gatsby-site/src/components/incidents/IncidentEditModal.js

```diff
@@ -84,6 +92,13 @@ export default function IncidentEditModal({ show, onClose, incidentId }) {
         createEntityMutation
       );
 
+      updated.epoch_date_modified = getUnixTime(new Date());
+
+      // Add the current user to the list of editors
+      if (user && user.providerType != 'anon-user' && !updated.editors.link.includes(user.id)) {
```

Comment:
Could this logic (and the `transformIncidentData` call)  be done in one function? Maybe create a custom hook `useLogHistory` that returns a function and then we have to only add one line everywhere we want to log a change:

```
const logStuff = useLogHistory();

... 

await logStuff({ ...everything needed here...)
```

Reply:
Issue https://github.com/responsible-ai-collaborative/aiid/issues/2138 added